package.json

```diff
@@ -13,9 +13,10 @@
     "retext"
   ],
   "dependencies": {
+    "hunspell-spellchecker": "^1.0.2",
+    "lodash": "^4.14.1",
```

Comment:
I kinda prefer the lone function modules over the whole lib, e.g., `lodash.includes`, instead of `lodash`.